### ISSUE TITLE
Icon colors (High Contrast)

### ISSUE THREAD

**Issue by ethan-fang-MS** [microsoft/devhome#1753]
### Windows build number

10.0.25977.0

### Steps to reproduce the bug

Turn on high contrast (Night Sky) and open Dev Home

### Expected result

All icons are either yellow or white, not a mix.

### Actual result

The Settings icon is yellow. The extensions icons within the extensions page are yellow. The other navigation icons are white.

![image](https://github.com/microsoft/devhome/assets/117125208/11a84b8e-8d0d-4af6-b3e6-a7fb14276fb3)

**Comment by ethan-fang-MS** [comment]
Prevailing thought is that all icons should be yellow.
